.github/workflows/deploy_staging_TestNet_UI.yml

```diff
@@ -22,7 +22,7 @@ jobs:
         run: echo ::set-output name=VERSION::${GITHUB_REF/refs\/tags\//}
 
       - name: Checkout
-        uses: actions/checkout@v3
+        uses: actions/checkout@v4
```

Comment:
should be 
`uses: actions/checkout@163217dfcd28294438ea1c1c149cfaf66eec283e # v4.2.1`